### PR TITLE
Adds 10 Chocolate Bars

### PR DESCRIPTION
**Pull request by BurgerLUA**
# Overview
Adds 10 chocolate bars, which are parodies of real life chocolate bars.

Swag

### CODE REVIEW

**Comment by BotBOREALIS** [comment]
Icons with diff:
<details><summary>icons/obj/food.dmi</summary>

Key | Old | New | Status
--- | --- | --- | ---
cb01||cb01|Created
cb02||cb02|Created
cb03||cb03|Created
cb04||cb04|Created
cb05||cb05|Created
cb06||cb06|Created
cb07||cb07|Created
cb08||cb08|Created
cb09||cb09|Created
cb10||cb10|Created
</details>

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -5242,5 +5242,156 @@
...
+	bitesize = 3
+	nutriment_type = NUTRIMENT_BAD
+	w_class = 1
+
+/obj/item/weapon/reagent_containers/food/snacks/cb08/Initialize()
+	. = ..()
+	reagents.add_reagent("sugar", 2)
+
+/obj/item/weapon/reagent_containers/food/snacks/cb09
+	name = "owo daddy bar"
```
Comment (Kaedwuff) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: N o

Pick another name for this bar. >:C

**Comment by Kaedwuff** [comment]
Oh, I suppose if it's just a halloween month gag that's not so bad.

**Comment by BurgerLUA** [comment]
gag uwu

**Pull request merged**